node/cli/src/runtime_version_cmd.rs

```diff
@@ -0,0 +1,48 @@
+use clap::Parser;
+use sc_cli::{CliConfiguration, Error, RuntimeVersion, SharedParams};
+use serde_json::{json, to_writer};
+use std::{fmt::Debug, fs, io, path::PathBuf};
+
+/// The `export-metadata` command used to export chain metadata.
+#[derive(Debug, Clone, Parser)]
+pub struct ExportRuntimeVersionCmd {
+	/// Output file name or stdout if unspecified.
+	#[clap(value_parser)]
+	pub output: Option<PathBuf>,
+
+	#[allow(missing_docs)]
+	#[clap(flatten)]
+	pub shared_params: SharedParams,
+}
+
+impl ExportRuntimeVersionCmd {
+	/// Run the get-runtime-version command.
+	pub async fn run(&self) -> Result<(), Error> {
+		let runtime_version: RuntimeVersion = self.read_runtime_version().unwrap();
+		let result = json!(runtime_version);
+		let file: Box<dyn io::Write> = match &self.output {
+			Some(filename) => Box::new(fs::File::create(filename)?),
+			None => Box::new(io::stdout()),
+		};
+		to_writer(file, &result).map_err(|_| Error::from("Failed Encoding"))
+	}
+
+	#[allow(unreachable_code)]
+	pub fn read_runtime_version(&self) -> Result<RuntimeVersion, Error> {
```

Comment:
can we use?
https://github.com/LibertyDSNP/frequency/pull/709/files#diff-ced5116f173c1d100d5f2d15b398a258b5c129b3eb8acbc9fb6e6ea876e9de79L205

Reply:
@enddynayn  cant seem to find that diff

Reply:
But I believe you are referring to native_runtime_version?

We cant access that without invoking runtime api which gives access to runtime via api which we dont need for this case, as we can read directly from runtime gives those values are static

Reply:
I think i found another approach

Reply:
Check now